config/groovy/common.groovy

```diff
@@ -215,8 +233,41 @@ class common {
                 println color("uncommitted changes. Skipping.", Ansi.YELLOW)
             } else {
                 println color("updating $itemType $itemName", Ansi.GREEN)
+                File targetDirFetchHead = new File("$targetDir/.git/FETCH_HEAD")
+                Date lastUpdate = new Date(targetDirFetchHead.lastModified())
+                def recentlyUpdated = use(groovy.time.TimeCategory){
+                    def timeElapsedSinceUpdate = new Date() - lastUpdate
+                    if (timeElapsedSinceUpdate < 2.minutes){
+                        println color("Skipping update for $itemName: updated within last 2 minutes", Ansi.YELLOW)
+                        return true
+                    } else {
+                        return false
+                    }
+                }
+                if (recentlyUpdated){
+                    return
+                }
```

Comment:
Honestly, I'd throw that out as I don't see any good reason for doing this :monocle_face: The solution works nicely, I just don't think we should do it that way...

Reply:
I was kind of hoping you would say that  :) I'll back this out.

Reply:
reverted